changelogs/fragments/2008-update-java-cert-replace-cert-when-changed.yml

```
@@ -0,0 +1,7 @@
+minor_changes:
+  - "Java_cert: Change `state: present` to check certificates by hash, not just alias name (https://github.com/ansible/ansible/issues/43249)"
+  - "Java_cert: cosmetic - modify run_command indentations"
+  - "Java_cert: remove password from run_command arguments as suggested by @felixfontein (https://github.com/ansible-collections/community.general/pull/2008)"
```

```suggestion
  - "java_cert - remove password from ``run_command`` arguments (https://github.com/ansible-collections/community.general/pull/2008)."
```
Also this should be in a `security_fixes` section.